wandb/env.py

```diff
@@ -161,6 +161,12 @@ def is_debug(default: Optional[str] = None, env: Optional[Env] = None) -> bool:
     return _env_as_bool(DEBUG, default=default, env=env)
 
 
+def is_offline(env: Optional[Env] = None) -> bool:
```

Comment:
nit: I realize this is consistent with existing code here, but `Optional` is redundant, as `Env` is already a type alias for `Optional[MutableMapping]`

```suggestion
def is_offline(env: Env = None) -> bool:
```

Reply:
Good eye, though looking at the current convention it seems to be fitting. I'll put another ticket to modify these at a later date.